testsuite/unittests/test_aeif_cond_beta_multisynapse.sli

```diff
@@ -0,0 +1,162 @@
+/*
+ *  test_aeif_cond_beta_multisynapse.sli
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+ 
+ /* BeginDocumentation
+   Name: testsuite::test_aeif_cond_beta_multisynapse - sli script for the aeif multisynapse model with synaptic conductance modeled by a double exponential function
+
+   Synopsis: (test_aeif_cond_beta_multisynapse) run
+
+   Description:
+   This test creates a multisynapse neuron with three receptor ports with
+   different synaptic rise times and decay times, and connect it to
+   two excitatory and one inhibitory signals. If the rise times and decay
+   times are much greater than the membrane time constant, the difference
+   between the membrane potential and the rest potential V(t) will be
+   approximately proportional to the synaptic conductance g(t), which can
+   be computed analitically. At the end, the script compares the simulated
+   values of V(t) with the approximate theoretical prediction.
```

Comment:
Could you add a derivation (or reference to a published derivation) of the approximation you use as reference?